### PR TITLE
Include SRI hashes for BokehJS in user guide section

### PR DESCRIPTION
**Pull request by santos22**
- [x] issues: fixes https://github.com/bokeh/bokeh/issues/9982

First time contributor, Season of Docs hopeful 👋
- I referenced js_resources.html for the code changes
- Fixed a small typo in js_resources.html 

Do we need to write tests for doc changes?

### CODE REVIEW

**Comment by mattpap** [comment]
Test failure is unrelated (is was marked as flaky in another PR).

File: bokeh/sphinxext/_templates/bokehjs_html_template.html [hunk 1]
```diff
@@ -5,8 +5,11 @@
 <title>Complete Example</title>
 {% for css_file in css_files %}
 <link rel="stylesheet" href="{{css_file}}" type="text/css" />{% endfor %}
-{% for js_file in js_files %}
-<script type="text/javascript" src="{{js_file}}"></script>{% endfor %}
+{%- for js_file in js_files %}
+<script type="text/javascript" src="{{ js_file }}"
```
Comment (santos22) [inline bokeh/sphinxext/_templates/bokehjs_html_template.html:9]: What’s the rule on spacing in the curly braces? Can change this back to `{{js_file}}`.

Comment (bryevdv) [reply bokeh/sphinxext/_templates/bokehjs_html_template.html:9]: I don't know what that we have any specific preference expressed, I'm personally fine with adding the spaces

**Comment by bryevdv** [comment]
Restarted to see if the flaky test will go green for good measure, but I will merge after CI is done regardless. Thanks @santos22 !

**Pull request merged**